src/index.js

```diff
@@ -79,9 +80,28 @@ $('.answer-card').on('click', function() {
 
 
 $('#submit-form__submit-btn').on('click', function() {
+  checkCardFlip()
   startTurn()
+  $('#submit-form__answer-input').val('');
+  checkRoundHighlight()
+  console.log(round.answers)
 })
 
+function checkCardFlip() {
+  round.answers.find
+  if ($('#submit-form__answer-input').val().toLowerCase() === 
```

Comment:
Since `$('#submit-form__answer-input').val().toLowerCase()` is repeated within the if/else statement, you can DRY this up and move it outside of the if/else to a variable and reference the variable.